analyzer/codechecker_analyzer/analyzers/config_handler.py

```diff
@@ -210,23 +212,40 @@ def initialize_checkers(self,
         profiles = checker_labels.get_description('profile')
         guidelines = checker_labels.occurring_values('guideline')
 
+        templ = Template("The ${entity} name '${identifier}' conflicts with a "
+                         "checker(-group) name. Please use -e ${entity}: to "
```

Comment:
We should be very specific about what conflicts with what. So if you can provide an exact error message instead of `...with a checker(-group)...` then please be specific.

Reply:
That's right, AFAIK it is always a checker group name that is conflicting, so I'll correct this.